cmd/nodecmd/create.go

```diff
@@ -814,6 +819,9 @@ func addNodeToClustersConfig(network models.Network, nodeID, clusterName string,
 		clustersConfig.Clusters = make(map[string]models.ClusterConfig)
 	}
 	if _, ok := clustersConfig.Clusters[clusterName]; !ok {
+		if isMonitoringInstance {
+			return fmt.Errorf("cluster %s is not found in cluster_config.json", clusterName)
```

Comment:
not sure if error message is correct.  we check ifMonitoringInstance but return err for cluster not found